src/view3d.cpp

```diff
@@ -448,92 +650,205 @@ void View3D::drawCursor(const TransformState& transStateIn, const V3d& cursorPos
     TransformState transState = transStateIn.translate(offset);
     V3d relCursor = cursorPos - offset;
 
-    // Cull if behind camera
-    if((relCursor * transState.modelViewMatrix).z > 0)
-        return;
+    // Cull if behind camera -> doesn't work for now
+    //if((relCursor * transState.projMatrix).z > 0)
+    //    return;
```

Comment:
True, it doesn't currently render behind the camera unless you use the undocumented ctrl+drag which lets you move it freely... but that's admittedly a bit useless as it stands and will be changed to something else at some stage.